src/ScanPatron/ScanForm.js

```diff
@@ -70,6 +136,10 @@ ScanForm.propTypes = {
   handleSubmit: PropTypes.func,
   handleScanPatron: PropTypes.func,
   form: PropTypes.object,
+  scannedPatronDetails: PropTypes.object,
+  patronRRAPermission: PropTypes.object,
+  resources: PropTypes.object,
+  resetDetails : PropTypes.func,
```

Comment:
Could you add defaultProps for those properties?

Reply:
I do not think default props are needed here too.